### PR TITLE
Edit fonts on Onboarding Screen

### PR DESCRIPTION
**Pull request by AndreiEgorov**
Editing fonts and Text positioning

### CODE REVIEW

File: js/screens/Onboarding/Onboarding.js [hunk 1]
```diff
@@ -50,73 +44,72 @@ let deviceWidth = Dimensions.get('window').width;
...
-          },
-          {
-            backgroundColor: '#76D2D1',
-            image: (
-              <Image
-                style={{ height: 219, width: 219 }}
-                source={require('../../assets/login/provider.png')}
-              />
-            ),
+    <Onboarding
```
Comment (siddharthparmar7) [inline js/screens/Onboarding/Onboarding.js:37]: Please move all your inline styling from here and put it in the styles.js file. Thank you.

File: js/screens/Onboarding/Onboarding.js [hunk 2]
```diff
@@ -50,73 +44,72 @@ let deviceWidth = Dimensions.get('window').width;
...
+          image: (
+            <Image
+              style={{ height: 219, width: 219 }}
+              source={require('../../assets/login/tutorial.png')}
+            />
+          ),
+          title: <Title title="View updates on your child" />,
+          subtitle: <Subtitle subtitle="Description to be added" />,
+          bottomBarHeight: 100,
+        },
```
Comment (siddharthparmar7) [inline js/screens/Onboarding/Onboarding.js:65]: Looks like there are a lot of similarities between your pages markup here. Please make a component and do the markup once only.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the requested changes. Thank you.

File: js/screens/Onboarding/Onboarding.js (hunk 2 above)
Comment (AndreiEgorov) [reply js/screens/Onboarding/Onboarding.js:65]: The background colour on onBoarding page will be assigned after the GlobalStyles colours are defined. (Will be with Cody's PR)

**Review by siddharthparmar7 — APPROVED** [review]
Thank you!

**Pull request merged**